### ISSUE TITLE
Sending SIGINT to giph doesn't do anything

### ISSUE THREAD

**Issue by lucypero** [phisch/giph#21]
If I call giph from a script, then `giph --stop` doesn't work, so I tried sending it the SIGINT signal but that doesn't do anything.

`pgrep -f giph | xargs kill -SIGINT`

**Comment by sudormrfbin** [comment]
Running `giph --stop` from a script doesn't work for me either:

```
/usr/bin/giph: line 474: kill: (-592148) - No such process
```

**Comment by phisch** [comment]
I have no idea what causes this, and i can't reproduce it. So if you can reproduce it, please find out why pgrep returns a process id that can't be killed by kill.

**Comment by phisch** [comment]
Closing this due to inactivity and it isn't reproducible for me.

**Comment by viktordanov** [comment]
I've just experienced the same problem and after some investigation, it appears it only works when the `giph` command is run in a bash environment. 
On line 474 you look for a process started by bash (I assume).
https://github.com/phisch/giph/blob/0198c9e73540db718a5d9f9637b586cc89932ca3/src/giph#L473 
In my case I use zsh as a daily-driver and use dash as a script interpreter so it wouldn't work either way. But when I tried running the commands from bash specifically, it worked. 

P.S. I'm using `pkill -f ffmpeg -2` to send SIGINT 
**P.P.S After more investigation, this isn't the problem but it's definitely something bash related**
